vividus-plugin-web-app/src/test/java/org/vividus/bdd/steps/ui/web/WaitStepsTests.java

```diff
@@ -441,4 +477,61 @@ void testWaitAlertDoesNotAppear()
         waitSteps.waitAlertDoesNotAppear(timeout);
         verify(softAssert).assertFalse("Alert does not appear", waitResult.isWaitPassed());
     }
+
+    @Test
+    void testWaitForElementDisappearanceBy()
+    {
+        By by = By.xpath(XPATH);
+        IExpectedSearchContextCondition<Boolean> condition = mock(IExpectedSearchContextCondition.class);
+        when(expectedSearchContextConditions.invisibilityOfElement(by)).thenReturn(condition);
+        WaitResult<Boolean> waitResult = new WaitResult<>();
+        when(waitActions.wait(webElement, condition)).thenReturn(waitResult);
+        waitResult.setWaitPassed(true);
+        assertTrue(waitSteps.waitForElementDisappearance(webElement, by));
+    }
+
+    @Test
+    void testWaitForElementDisappearanceWithAttributes()
+    {
+        SearchAttributes attributes = new SearchAttributes(ActionAttributeType.ELEMENT_NAME, NAME);
+        IExpectedSearchContextCondition<Boolean> condition = mock(IExpectedSearchContextCondition.class);
+        when(expectedSearchActionsConditions.invisibilityOfElement(attributes)).thenReturn(condition);
+        WaitResult<Boolean> waitResult = new WaitResult<>();
+        when(waitActions.wait(webElement, condition)).thenReturn(waitResult);
+        waitResult.setWaitPassed(true);
+        assertTrue(waitSteps.waitForElementDisappearance(webElement, attributes));
+    }
+
+    @Test
+    void testWaitForElementDisappearanceWithDuration()
+    {
+        SearchAttributes attributes = new SearchAttributes(ActionAttributeType.ELEMENT_NAME, NAME);
+        IExpectedSearchContextCondition<Boolean> condition = mock(IExpectedSearchContextCondition.class);
+        when(expectedSearchActionsConditions.invisibilityOfElement(attributes)).thenReturn(condition);
+        WaitResult<Boolean> waitResult = new WaitResult<>();
+        when(waitActions.wait(webElement, TIMEOUT, condition)).thenReturn(waitResult);
+        waitResult.setWaitPassed(true);
+        assertTrue(waitSteps.waitForElementDisappearance(webElement, attributes, TIMEOUT));
+    }
+
+    @Test
+    void testWaitForElementDisappearanceByWithDuration()
+    {
+        By by = By.xpath(XPATH);
```

Comment:
why not to extract it to separate static field?

Reply:
it's used several times, but not all tests require this field